REFERENCE.md

```diff
@@ -2689,15 +2689,15 @@ Copy QUEUE as another queue.
 
 ## Box
 
-### `(unbox box)`
```

Comment:
Is this sort of order instability in the documentation generator expected? @ruricolist